rpm/python3-babel.spec

```diff
@@ -0,0 +1,43 @@
+Summary:       Babel
+Name:          python3-babel
+Version:       0
+Release:       1
+License:       BSD and Unicode
+Source0:       %{name}-%{version}.tar.gz
+Source1:       cldr-common-41.0.zip
+URL:           https://github.com/sailfishos/python3-babel
+
+BuildRequires: python3-devel
+BuildRequires: python3-setuptools
+BuildRequires: python3-pytz
+
+%description
+Python Internationalization Library
+
+%package -n babel
+Summary: Tools for internationalizing Python applications
+Requires: %{name} = %{version}-%{release}
+
+%description -n babel
+Tools for internationalizing Python applications
+
+%prep
+%autosetup -n %{name}-%{version}/%{name}
+
+%build
+cp ../rpm/cldr-common-41.0.zip cldr/
+%python3 ./setup.py import_cldr
+%py3_build
+
+%install
+%py3_install
+
+%files
+%defattr (-,root,root,-)
+%license LICENSE cldr/cldr-common-41.0/LICENSE.txt
```

Comment:
Maybe use wildcard for easier maintenance?

Reply:
Which part would you replace with wildcard? The cldr version number?

Reply:
yup